.drone.yml

```diff
@@ -12,7 +12,7 @@ steps:
     commands:
       - grep -vE '^\s*(#|//)' images-list | sort -Vc
   - name: mirror-images
-    image: rancher/dapper:v0.5.3
+    image: rancher/dapper:v0.5.7
```

Comment:
This looks fine to me but IIRC anything past dapper 0.5.0 segfaults on arm (i am not super familiar with this repo and what it does, actually)

Reply:
That shouldn't actually matter, we pull/push the manifest based on the OS's specified in the line changed below, not what OS you are currently running on.

Reply:
I am curious though why we upgraded, if there was something new in dapper we needed to pull in, or if this is just a generic upgrade ?

Reply:
we need to upgrade to 0.5.7 because 0.5.7 has an s390x arch image published to dockerhub while 0.5.3 doesn't.
Without it, the s390x images won't be built properly.

Reply:
@cjellick  there are 2 mirrored images in the list that are being used by rke2:
mirrored-pause  and mirrored-ingress-nginx-kube-webhook-certgen